### PR TITLE
website: add community governance, etc.

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Preview: https://deploy-preview-2353--lima-vm.netlify.app/docs/community/
- - -
Establish the governance model so as to facilitate getting more contributions from new and existing contributors.

I also expect that this will help getting more involvements from other maintainers for reviewing/merging PRs and making new releases. Too much power and reponsibility have been concentrating on me.

This PR is intended to fulfill the requirements listed in [CNCF Incubation Application v1.5](https://github.com/cncf/toc/blob/8211f562bb3123a7be85c2de4f3b741a0e448e66/.github/ISSUE_TEMPLATE/template-incubation-application.md).

See also ["Applying to become an Incubating or Graduating project"](https://github.com/cncf/toc/tree/8211f562bb3123a7be85c2de4f3b741a0e448e66/process#applying-to-become-an-incubating-or-graduating-project)

- [X] Governance
- [X] Release process
- [X] Contribution guide
- [X] Subproject list

TODOs:
- Complete [OpenSSF Best Practices](https://www.bestpractices.dev/en/projects/6505)
- Submit [Security Self-Assessment](https://github.com/cncf/tag-security/blob/main/assessments/guide/self-assessment.md) to <https://github.com/cncf/tag-security/tree/main/assessments/projects>

- Submit an incubation application to <https://github.com/cncf/toc/issues>

### CODE REVIEW

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [inline website/content/en/docs/Community/Governance/_index.md:36]: @lima-vm/maintainers Please add your GPG fingerprints, so that you all can be eligible to be a release manager (see the "Release process" section)

Comment (afbjorklund) [reply website/content/en/docs/Community/Governance/_index.md:37]: Apparently it has been a while since I gave up on signing... `pub   rsa2048 2018-06-10 [SC] [expired: 2019-06-10]`

Comment (jandubois) [reply website/content/en/docs/Community/Governance/_index.md:37]: Similar for me: `pub   rsa4096 2017-03-23 [SC] [expired: 2023-04-14]`.

I assume we can create a new key and upload it, even if it is not signed by other people?

Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md:37]: You can just renew expired keys https://filipe.kiss.ink/renew-expired-gpg-key/

Comment (balajiv113) [reply website/content/en/docs/Community/Governance/_index.md:37]: I have been using SSH signing key. I can create a new GPG key and upload it

Comment (afbjorklund) [reply website/content/en/docs/Community/Governance/_index.md:37]: Will try to find and renew, otherwise will replace it.

File: website/content/en/docs/Community/Governance/_index.md
Comment (jandubois) [inline website/content/en/docs/Community/Governance/_index.md:69]:
```suggestion
- MUST be an active Committer
```

File: website/content/en/docs/Community/Governance/_index.md
Comment (jandubois) [inline website/content/en/docs/Community/Governance/_index.md:50]: How are new maintainers added to the CNCF list?

File: website/content/en/docs/Community/Governance/_index.md
Comment (jandubois) [inline website/content/en/docs/Community/Governance/_index.md:48]: Should there be a quorum for any of the voting, or is a single vote enough if nobody else votes in the time period?

File: website/content/en/docs/Community/Governance/_index.md
Comment (jandubois) [inline website/content/en/docs/Community/Governance/_index.md:64]: Is there a tie-breaker, or is any tie a failed vote?

**Review by jandubois — APPROVED** [review]
Looks mostly fine to me (1 typo).

The feedback on voting is just nit-picking based on role-playing experience with rules-lawyers, and probably not really important here. 😄

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md:48]: Maybe it should need 2 approvals (including the proposer) at least?

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md:64]: I guess it should be considered a failed vote?
Basically we should try to reach consensus

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md:50]: With a PR like this
- https://github.com/cncf/foundation/pull/500

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md:37]: Thanks, added

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md]: fixed

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md:48]: Added "Voting needs 2 approvals at least. The proposer can vote too."

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md:66]: Added "A tie should be considered as a failed vote."

**Review by jandubois — APPROVED** [review]

File: website/content/en/docs/Community/Contributing/_index.md
Comment (jandubois) [inline website/content/en/docs/Community/Contributing/_index.md:29]: Since I just merged my own PR I'm wondering: shouldn't this say that "committers shouldn't merge their own pull request without approval by at least one other committer"?

I don't think the merging is the issue; it is merging without independent review.

File: website/content/en/docs/Community/Governance/_index.md
Comment (afbjorklund) [reply website/content/en/docs/Community/Governance/_index.md:37]: New key:
`5981 D2E8 4E4B 9197 95B3  2174 DC05 CAD2 E73B 0C92`

https://github.com/afbjorklund.gpg

File: website/content/en/docs/Community/Contributing/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Contributing/_index.md]: Reworded
```markdown
## Merging pull requests

[Committers](../governance) can merge pull requests.
[Reviewers](../governance) can approve, but cannot merge, pull requests.

A Committer shouldn't merge their own pull requests without approval by at least one other Maintainer (Committer or Reviewer).

This rule does not apply for trivial pull requests such as fixing typos, CI failures,
and updating image references in templates (e.g., <https://github.com/lima-vm/lima/pull/2318>).
```

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md]: Thanks, added

File: website/content/en/docs/Community/Governance/_index.md
Comment (jandubois) [inline website/content/en/docs/Community/Governance/_index.md:20]:
```suggestion
  Committers' commits should still be made via GitHub pull requests (except for urgent security fixes), and should not be pushed directly.
```

File: website/content/en/docs/Community/Governance/_index.md
Comment (jandubois) [inline website/content/en/docs/Community/Governance/_index.md:25]:
```suggestion
  A Reviewer is considered as a candidate to become a Committer.
```

**Review by jandubois — DISMISSED** [review]
I've only re-reviewed the Governance document. One typo and one wording nit-pick.

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md]: Thanks, fixed

File: website/content/en/docs/Community/Governance/_index.md
Comment (AkihiroSuda) [reply website/content/en/docs/Community/Governance/_index.md]: Thanks, fixed

**Review by jandubois — APPROVED** [review]
I think this PR is ready for merging now.

**Comment by AkihiroSuda** [comment]
@afbjorklund @balajiv113 PTAL 🙏

**Review by afbjorklund — APPROVED** [review]

**Review by balajiv113 — APPROVED** [review]

**Pull request merged**
